static/sass/_pattern_cve.scss

```diff
@@ -31,6 +40,70 @@
     padding-left: 0.75rem;
   }
 
+  .cve-table .icon-cve-info {
+    margin: 0 0 0 0.5rem;
+  }
+
+  .cve-table .icon-container__text {
+    width: calc(100% - 3rem);
+  }
+
+  .cve-table .p-icon--information {
+    opacity: 0.6;
+  }
+
+  .cve-table-cell .p-icon--information {
+    visibility: hidden;
+  }
+
+  .cve-table-cell:hover .p-icon--information {
+    visibility: visible;
+  }
+
+  .cve-table-cell .cve-tooltip {
+    display: none;
+  }
+
+  .cve-table-cell:hover .p-icon--information:hover + .cve-tooltip {
+    display: block;
+  }
+
+  .cve-table .icon-container__icon {
+    position: relative;
+  }
+
+  .cve-tooltip {
+    z-index: 1;
+    background: #fff;
+    position: absolute;
+    top: 30px;
+    left: -125px;
```

Comment:
Instead of this you could do: `transform: translateX(-125px)`. It's better for in-page performance.